lib/StackageToHackage/Hackage.hs

```diff
@@ -61,15 +65,23 @@ import qualified Data.Map.Strict as M
 import qualified Data.Text as T
 import System.Directory (doesFileExist)
 
+ -- | Whether to inspect remotes.
+newtype InspectRemotes = InspectRemotes Bool
```

Comment:
What's the point? 

This is not exposed API.

Reply:
I didn't want to add another argument to functions with a signature `:: Bool -> _` that would then become `:: Bool -> Bool -> _` so used `newtype` to distinguish these `Bool` arguments.

Reply:
Don't see the value. You can create a record type if you think it's too many args.

Reply:
Before going full naked boolean as you suggest, do you find putting those newtypes into the option record and using coerce instead of explicit unpacking any better?

Reply:
No. I don't want newtypes for something this simple.